epam_tr/src/main/java/com/epam/tr/dto/FileDto.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.tr.dto;
+
+import java.util.Objects;
+
+public class FileDto {
+
+    public enum FileSystemObjectType {
```

Comment:
it is usually placed in the end of the class.
also you did change the name of the enum...